java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -256,6 +261,10 @@ class DatabaseImpl(
         val singletons = mutableMapOf<FieldName, Referencable?>()
         val collections = mutableMapOf<FieldName, MutableSet<Referencable>>()
 
+        val numeric_types = "(${PrimitiveType.Number.ordinal}, " +
+            "${PrimitiveType.Float.ordinal}, " +
+            "${PrimitiveType.Double.ordinal})"
```

Comment:
Another option would be to use `joinToString`:

```
listOf(
    PrimitiveType.Number.ordinal,
    PrimitiveType.Float.ordinal,
    PrimitiveType.Double.ordinal
).joinToString(prefix="(", postfix=")")
```

Reply:
yeah that looks better. Updated here and in clear.